R/resolve-links.R

```diff
@@ -0,0 +1,112 @@
+#' Resolve Reference-Style Links
+#'
+#' Commonmark treats reference-style links as regular links, which can be a
+#' pain when converting large documents. This is an attempt at resolving these
+#' links by reading in the source document, finding the relative links, and 
+#' adding them back with the 'asis' attribute.
+#'
+#' @param body an XML body
+#' @param path path to the source file
+#' @param ns an the NS that resolves the md ns
+#' @param encoding the encoding of the file. Defaults to UTF-8
```

Comment:
are these parameters actually used?

Reply:
Oh! good catch! The doc was written when I was still using the path as the input instead of the lines of text. I will fix this!